modules/tabs-v2/assets/scss/frontend.scss

```diff
@@ -131,15 +161,6 @@
 		}
 	}
 
-	.elementor-tab-title,
```

Comment:
There was a lot of code in the pr from the original tabs widget
I have removed a lot of code, to avoid chaos. If we need this styling, we will find it out in the tabs styling tasks.

Reply:
The element still exists in the V2 Tabs markup... Did you check if it is affected by removing it?

Reply:
Just to be safe, it will only only remove the 'tab-title' classes (because that is the focus of this PR) and I will keep the styling for the other classes.

'elementor-tab-title' I had styled in this PR.